client/src/main/java/org/spine3/protobuf/EntityPackagesMap.java

```diff
@@ -62,7 +62,7 @@
      * <p>Example:
```

Comment:
Please have a paragraph after abstract line of the property. Then you'll be able to have `Example:` in the same paragraph.